ci/README.md

```diff
@@ -41,7 +41,7 @@ jobs:
     - stage: build
       script: ./build.sh     # build and push docker images
     - stage: deploy
-      script: ./ketch-ci.sh --ketch-tag v0.1.1 -a myapp -o mypool --endpoint 104.155.134.17 -i docker.io/shipasoftware/bulletinboard:1.0 --ingress traefik
+      script: ./ketch-ci.sh --ketch-tag v0.1.1 -a myapp -o myframework --endpoint 104.155.134.17 -i docker.io/shipasoftware/bulletinboard:1.0 --ingress traefik
```

Comment:
shipa uses -k and --framework to specify a framework, I think we can follow the same convention
```
-k, --framework (= "")
```

Reply:
Modified to `-k` in readme, ketch-ci script, and short flags (deploy/parameters.go).